tests/comments/trailing_space.js

```diff
@@ -3,3 +3,8 @@
 // Do not trim trailing whitespace from this source file!
 
 // There is some space here ->                        
+
+var a = { /* extra whitespace --> */  
+  b };
+var a = { /* no whitespace --> */
+  b };
```

Comment:
Please create a new test file. Don't modify existing ones.

Reply:
Thanks. I created a new test file. Please check it.